jsonrpc/filter_manager.go

```diff
@@ -15,57 +16,61 @@ import (
 	"github.com/hashicorp/go-hclog"
 )
 
-type Filter struct {
-	id string
+var ErrFilterDoesNotExists = errors.New("filter does not exists")
 
-	// block filter
-	block *headElem
+// defaultTimeout is the timeout to remove the filters that doesn't have web socket stream
```

Comment:
Typo here: `that don't have a web socket stream`

Reply:
Fixed at b0728d05324cea09366d9761c7661205e4e6dcac